### PR TITLE
version 8.0.3

### PR DESCRIPTION
**Pull request by szymonlesisz**
# 8.0.3
#### Added
- Added `Binance Chain (BNB)` support
#### Fixed
- `TrezorConnect.cancel` race condition between device release and returned response
#### Updated 
- protobuf messages

### CODE REVIEW

File: src/__tests__/core/binanceSignTransaction.spec.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+/* @flow */
+
+import type {
+    TestFunction,
+    SubtestSignTransaction,
+} from 'flowtype/tests';
+
+import type {
+    TestCardanoSignTransactionPayload,
```
Comment (prusnak) [inline src/__tests__/core/binanceSignTransaction.spec.js]: This looks weird - importing CardanoTypes into Binance code?

File: Makefile [hunk 2]
```diff
@@ -50,7 +50,7 @@ sync-%:
 # Build messages.json from protobuf
 protobuf:
 	make -C ./submodules/trezor-common/protob combine
-	./node_modules/.bin/proto2js ./submodules/trezor-common/protob/combined.proto > ./src/data/messages.I.json
+	./node_modules/.bin/proto2js ./submodules/trezor-common/protob/combined.proto > ./src/data/messages/messagesN.json
```
Comment (prusnak) [inline Makefile:53]: Why this change?

Comment (szymonlesisz) [reply Makefile:53]: messages are now placed in "messages" dir not "data" dir

File: src/__tests__/core/binanceSignTransaction.spec.js (hunk 1 above)
Comment (szymonlesisz) [reply src/__tests__/core/binanceSignTransaction.spec.js]: solved, thanks
